src/Projects.tsx

```diff
@@ -13,7 +13,7 @@ class Project extends React.Component<Props, State> {
     var data = this.props.project;
     var roles;
     if (data.lookingFor && data.lookingFor.length > 1) {
-      roles = data.lookingFor[0] + ', ' + data.lookingFor[1];
+      roles = data.lookingFor[0] + ',  ' + data.lookingFor[1];
```

Comment:
Is this intentional? Looks like the original version should be fine. Styling should be done with css.